setup_machine.sh

```diff
@@ -40,9 +44,20 @@ cd pythonapp-createjazzlyric
 # python3.9 -m venv .venv
 pipenv install
 
+# FLASK_ENV - by default, is production, which doesn't do anything noticeable.
+# development - see reloader starts working & your app is put into debug mode
+# Default port for Flask here is 5000. So if I don't set to anything else, IS 5000.
+# FLASK_RUN_PORT=8084
 cat > .flaskenv <<EOF
   FLASK_APP=create_jazz_lyric
   FLASK_RUN_HOST=0.0.0.0
 EOF
 
-pipenv shell
+if [ -f '../mysql_pw.txt' ]; then
+  {
+    echo 'DB_HOST=localhost'
+    echo 'DB_USER=root'
+    echo "DB_PASSWORD=$(cat ../mysql_pw.txt)"
+    echo 'DB_NAME=lyric_db'
+  } > .env
+fi
```

Comment:
Suggestion: if `../mysql_pw.txt` doesn't exist, you will probably want to fail the script with a message (I was being a bit sloppy when I suggested adding this part). You could do this by adding an `else` where you echo a message and invoke `exit 1`.

Reply:
Good point about failing early if the required file, `mysql_pw.txt`, is missing. Added `else` clause.

Reply:
Also, added a check for if Python 3.9 is already installed, which you mentioned several lines above, so the 7 lines don't need to be re-run:
```
if [[ $(python3.9 --version) != "Python 3.9.7" ]]; then
  echo "Getting and installing Python 3.9.7..."
  wget https://www.python.org/ftp/python/3.9.7/Python-3.9.7.tgz
  ...
```

Reply:
OK, you have addressed the issue.